site/public/js/notification-sound.js

```diff
@@ -0,0 +1,15 @@
+class NotificationSound {
+    constructor() {
+        this.sound = document.createElement('audio');
+
+        if (this.sound.canPlayType('audio/wav') === '') {
```

Comment:
it occurs to me that we probably want to convert this sound file into MP3, which would shrink the raw audio file from 168kb to 9kb (and giving us a similar reduction in this file). It looks like MP3 is well supported on all browsers we support, and that I doubt anyone will notice the very slight degradation of quality.